src/sentry/api/serializers/models/organization_member.py

```diff
@@ -53,8 +53,9 @@ def get_attrs(self, item_list, user):
 
         # results is a map of member id -> team_slug[]
         for m in member_team_map:
-            results[m['organizationmember_id']].append(
-                teams[m['team_id']].slug)
+            if teams[m['team_id']].status is TeamStatus.VISIBLE:
```

Comment:
deleted teams were showing up in this list and causing problems

Reply:
Just filter this out of the query in the first place.

```python
teams = {team.id: team for team in Team.objects.filter(
    status=TeamStatus.VISIBLE,
    id__in=[item['team_id'] for item in member_team_map])}
```

Or even inside the `OrganizationMemberTeam` query with `team__status=TeamStatus.VISIBLE`